src/internal/connector/exchange/data_collections.go

```diff
@@ -174,7 +173,8 @@ func DataCollections(
 	}
 
 	var (
-		scopes      = eb.DiscreteScopes(userPNs)
+		user        = selector.DiscreteOwner
+		scopes      = eb.DiscreteScopes([]string{user})
```

Comment:
I take it this will eventually be going away since we're already down to a single resource owner at this point?

Reply:
Yep!  One thing at a time.

Reply:
sounds good. Thanks for breaking this all into chunks for review